_studio/shared/include/mfx_mfe_adapter.h

```diff
@@ -27,10 +27,11 @@
 #include <map>
 #include <list>
 #include <vector>
-#include "vm_mutex.h"
-#include "vm_cond.h"
 #include "vm_time.h"
 #include <mfxstructures.h>
+#undef max
+#undef min
```

Comment:
Why you need these undefs? I am _not_ ok with such a code. This is a hack which should be understood and eliminated.

Reply:
The problem is conflict definition of macros "max" from chrono and cm_mem_copy.h which includes in mfx_vpp_hw.h.
Conflict definitions happens in mfx_vpp_hw.cpp: 
    mfx_vpp_hw.h <- cm_mem_copy.h
    linmfx_core_vaapi.h <- mfx_mfe_adapter.h <- condition_variable <- chrono